robottelo.properties.sample

```diff
@@ -202,6 +202,11 @@ ssh_key=
 # External docker registries urls in the format http[s]://<server>[:<port>].
 # external_registry_1=
 # external_registry_2=
+# For testing the support for private registry repos 
+# private_registry_url='https://registry.hub.docker.com'
+# private_registry_name=username/imagename
```

Comment:
could you change _name to be something more descriptive, like _image?

Reply:
I used `name` because the API calls it `docker_upstream_name`